great3sims/psf.py

```diff
@@ -1382,10 +1382,22 @@ class DrawPSFBuilder(PSFBuilder):
     def __init__(self, obs_type, multiepoch, shear_type, draw_psf_src):
         self.obs_type = obs_type
         self.draw_psf_src = os.path.abspath(draw_psf_src)
+        filename = os.path.basename(self.draw_psf_src)
```

Comment:
This code was written by me exclusively for this project.  And as our approach to providing access to the Psfs has changed, so has the code.  I don't anticipate having a version of the code again which does not have an index, and no one else is relying on it.

We are not planning to push this code back to great3, but I agree that it could use a few more line comments, so I will add them.